watcher/src/config.rs

```diff
@@ -52,7 +51,7 @@ impl WatcherConfig {
 
 /// A single watched source configuration.
 #[derive(Clone, Debug, Deserialize, Serialize, Eq, PartialEq, Hash)]
-struct SourceConfig {
+pub struct SourceConfig {
```

Comment:
It would be nice to extract this (and some other common stuff) into a `mc-common-config` crate...

Reply:
I tried it in the past but it's tricky, different applications want to store different things. For example, consensus-service might want to include SCP keys as part of the configuration (and make them mandatory) but the watcher doesn't care about that.

Reply:
Mmm, yeah.